LibsDisguises throws an exception on 1.18.2

```
[12:07:48 ERROR]: Error occurred while enabling LibsDisguises v10.0.27 (Is it up to date?)
java.lang.NoClassDefFoundError: org/bukkit/craftbukkit/v1_18_R1/inventory/CraftItemStack
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:375) ~[?:?]
        at me.libraryaddict.disguise.utilities.reflection.ReflectionManager.getReflectionManager(ReflectionManager.java:698) ~[LibsDisguises-10.0.27-Free.jar:?]
        at me.libraryaddict.disguise.utilities.reflection.ReflectionManager.init(ReflectionManager.java:123) ~[LibsDisguises-10.0.27-Free.jar:?]
        at me.libraryaddict.disguise.LibsDisguises.onEnable(LibsDisguises.java:218) ~[LibsDisguises-10.0.27-Free.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:501) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugin(CraftServer.java:559) ~[paper-1.18.2.jar:git-Paper-220]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugins(CraftServer.java:473) ~[paper-1.18.2.jar:git-Paper-220]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:666) ~[paper-1.18.2.jar:git-Paper-220]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:433) ~[paper-1.18.2.jar:git-Paper-220]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:316) ~[paper-1.18.2.jar:git-Paper-220]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1156) ~[paper-1.18.2.jar:git-Paper-220]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.2.jar:git-Paper-220]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ClassNotFoundException: org.bukkit.craftbukkit.v1_18_R1.inventory.CraftItemStack
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:151) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        ... 16 more
[12:07:48 INFO]: [LibsDisguises] Disabling LibsDisguises v10.0.27
```

--- a/nms/v1_18_R2/src/main/java/me/libraryaddict/disguise/utilities/reflection/v1_18_2/ReflectionManager.java
+++ b/nms/v1_18_R2/src/main/java/me/libraryaddict/disguise/utilities/reflection/v1_18_2/ReflectionManager.java
@@ -452,7 +452,7 @@ public class ReflectionManager implements ReflectionManagerAbstract {
 
     public ItemMeta getDeserializedItemMeta(Map<String, Object> meta) {
         try {
-            Class<?> aClass = Class.forName("org.bukkit.craftbukkit.v1_18_R1.inventory.CraftMetaItem$SerializableMeta");
+            Class<?> aClass = Class.forName("org.bukkit.craftbukkit.v1_18_R2.inventory.CraftMetaItem$SerializableMeta");
             Method deserialize = aClass.getDeclaredMethod("deserialize", Map.class);
             Object itemMeta = deserialize.invoke(null, meta);
 

